pyproject.toml

```diff
@@ -78,6 +78,7 @@ flask-cors = ">3.0.8"
 celery = "^5.0.0"
 lxml = ">=4.6.2"
 cryptography = ">3.2"
+py = "^1.10.0"
```

Comment:
`py` is not a direct dependency in our code. It is better to add a TODO with a reason why we added it , so we can delete this line later. Why the version must be fixed to `1.10.0`, Would it not be better if we say `>=1.10.0`?